### PR TITLE
Deploy the Telegraf-based connectors at TTS

### PR DESCRIPTION
**Pull request by afausti**
This PR deploys the Telegraf-based connectors at TTS using same configuration we tested at BTS.

### CODE REVIEW

File: applications/sasquatch/values-tucson-teststand.yaml [hunk 1]
```diff
@@ -76,122 +76,96 @@ influxdb:
...
+      debug: true
+    genericcamera:
       enabled: true
+      database: "efd"
       topicRegexps: |
-        [ "lsst.sal.Test" ]
-    genericcamera:
+        [ "lsst.sal.GCHeaderService", "lsst.sal.GenericCamera" ]
+      debug: true
+    lsstcam:
```
Comment (mareuter) [inline applications/sasquatch/values-tucson-teststand.yaml:163]: LSSTCam is not available on TTS. Remove it.

File: applications/sasquatch/values-tucson-teststand.yaml [hunk 2]
```diff
@@ -76,122 +76,96 @@ influxdb:
...
       topicRegexps: |
-        [ ".*Scheduler", ".*Script", ".*ScriptQueue", ".*Watcher" ]
+        [ "lsst.sal.Scheduler", "lsst.sal.Script", "lsst.sal.ScriptQueue", "lsst.sal.Watcher" ]
+      debug: true
     ocps:
       enabled: true
+      database: "efd"
       topicRegexps: |
-        [ ".*OCPS" ]
-    calsys:
```
Comment (mareuter) [inline applications/sasquatch/values-tucson-teststand.yaml:124]: The calibrations systems are still in Tucson. This needs to be restored.

File: applications/sasquatch/values-tucson-teststand.yaml [hunk 3]
```diff
@@ -76,122 +76,96 @@ influxdb:
...
       topicRegexps: |
-        [ ".*MTAOS", ".*MTDome", ".*MTDomeTrajectory", ".*MTPtg" ]
+        [ "lsst.sal.MTAOS", "lsst.sal.MTDome", "lsst.sal.MTDomeTrajectory", "lsst.sal.MTPtg" ]
+      debug: true
     mtmount:
       enabled: true
+      database: "efd"
       topicRegexps: |
-        [ ".*MTMount" ]
-    comcam:
```
Comment (mareuter) [inline applications/sasquatch/values-tucson-teststand.yaml:93]: ComCam is on TTS. This needs to be restored/added.

File: applications/sasquatch/values-tucson-teststand.yaml [hunk 4]
```diff
@@ -76,122 +76,96 @@ influxdb:
...
       topicRegexps: |
-        [ ".*OCPS" ]
-    calsys:
+        [ "lsst.sal.OCPS" ]
+      debug: true
+    test:
       enabled: true
+      database: "efd"
       topicRegexps: |
-        [ ".*ATMonochromator", ".*ATWhiteLight", ".*CBP", ".*Electrometer", ".*FiberSpectrograph", ".*LEDProjector", ".*LinearStage", ".*MTReflector", ".*TunableLaser" ]
```
Comment (mareuter) [inline applications/sasquatch/values-tucson-teststand.yaml:127]: ATMonochromator and ATWhiteLight can be removed.

**Review by mareuter — CHANGES_REQUESTED** [review]
Somethings to fix up here otherwise it will mess up TTS.

**Comment by afausti** [comment]
Argh, sorry about that, I naively copied the BTS configuration here.

**Review by mareuter — APPROVED** [review]
Diffs are a bit weird, but I think all the fixes are there.

**Pull request merged**